### ISSUE TITLE
Add Hammerjs to handle touch gesture - suggestion

### ISSUE THREAD

**Issue by Nightbr** [almasaeed2010/AdminLTE#357]
Hi everyone,
I think It would be cool if **hammerjs** will be integrated in AdminLTE. For example, I enhanced the PushMenu with swipe event in order to close/open the menu with right and left swipe.

Here is the code to put in the `$.AdminLTE.pushMenu` function:
``` javascript
// touch

  $('body').hammer().on('swiperight', function(e) {
      e.preventDefault();
      //OPEN
      console.log("swiperight");
      if ($(window).width() > (screenSizes.sm - 1)) {
        if($("body").hasClass('sidebar-collapse'))
        {
            $("body").removeClass('sidebar-collapse');
        } 
      }else{
        if (!$("body").hasClass('sidebar-open')) {
          $("body").addClass('sidebar-open');
        }
      }
   
  });
             
  $('body').hammer().on('swipeleft', function(e) {
      e.preventDefault();
      //CLOSE
      console.log("swipeleft");
      if ($(window).width() > (screenSizes.sm - 1)) {
        if(!$("body").hasClass('sidebar-collapse'))
        {
            $("body").addClass('sidebar-collapse');
        } 
      }else{
        if ($("body").hasClass('sidebar-open')) {
          $("body").removeClass('sidebar-open');
          $("body").removeClass('sidebar-collapse');
        }
      }
          
  });
```

And after that, you can handle other touch action with hammerjs.
Take a look at http://hammerjs.github.io/

Regards

**Comment by Nightbr** [comment]
Note1: I used the jQuery plugin of hammerjs: http://hammerjs.github.io/jquery-plugin/
So, in my index.html, I had to add both library:
``` HTML
<script src="bower_components/hammerjs/hammer.js" type="text/javascript"></script>
        <script src="bower_components/jquery-hammerjs/jquery.hammer.js" type="text/javascript"></script>
```
Note2: For the swipe menu example (see above), Swipe listeners prevents responsive table from moving right or left. So keep this in mind. In my case, I use dataTable-responsive plugins and all my tables are dataTables.

If you have any questions, don't hesitate to ask.
